protocols/user_data_processor.py

```
@@ -1,95 +1,103 @@
+                                    }
+                                )
+                            elif file.endswith(".txt"):
+                                processed_files.append(
+                                    {
+                                        "path": file_path.replace(path, f"{name}/"),
+                                        "type": "text",
+                                        "size": size,
+                                    }
+                                )
+                        except BaseException:
                             pass
```

Ruff (0.11.9)

`try`-`except`-`pass` detected, consider logging the exception

(S110)